eng/common/scripts/Update-DocsMsToc.ps1

```diff
@@ -218,13 +225,48 @@ foreach ($service in $serviceNameList) {
 }
 
 # Core packages belong under the "Other" node in the ToC
-$otherPackageItems = @()
+$otherPackageItems = New-Object -TypeName System.Collections.Generic.List[PSCustomObject]
 $otherPackages = $packagesForToc.Values.Where({ $_.ServiceName -eq 'Other' })
-$otherPackages = $otherPackages | Sort-Object -Property Package
+$otherPackages = $otherPackages | Sort-Object -Property DisplayName
 
 if ($otherPackages) {
   foreach ($otherPackage in $otherPackages) {
-    $otherPackageItems += GetClientPackageNode $otherPackage
+    $segments = $otherPackage.DisplayName.Split(' - ')
```

Comment:
Should we split only on "-" and then trim any whitespace? Just to make this a little more tolerable to simple typos.